### ISSUE TITLE
Feature request: keyboard shortcuts

### ISSUE THREAD

**Issue by curiositry** [Photosynthesis/Timetracker#7]
Are keyboard shortcuts on the roadmap? They can save a lot of time hunting and pecking for power users, without getting in the way of casual users who are just getting the hang of the app.

**Comment by VA7NFH** [comment]
I agree with curiositry, is there an easy way to add keyboard shortcuts?
